dialog/base.go

```diff
@@ -141,16 +155,16 @@ func newButtonList(buttons ...*widget.Button) fyne.CanvasObject {
 }
 
 func (d *dialog) Show() {
-	go d.wait()
+	d.wait()
```

Comment:
`wait` sounds like a synchronous call but now it runs a goroutine. Maybe it should be renamed to reflect this?